app/src/test/java/nl/tudelft/bbw/APITest.java

```diff
@@ -66,12 +76,12 @@ public final void setUp() throws HashException, BlockAlreadyExistsException {
     @Test
     public final void addContactToChainTest() throws HashException, BlockAlreadyExistsException,
             NoSuchAlgorithmException, InvalidKeyException, SignatureException {
-        API.initializeAPI(newUser, RuntimeEnvironment.application);
-        final byte[] message = owner.getPublicKey().getEncoded();
-        final byte[] signature = ED25519.generateSignature(message, newUser.getPrivateKey());
-        list = API.getBlocks(newUser);
-        API.addContactToChain(owner, signature, message);
-        assertNotEquals(API.getBlocks(newUser), list);
+
+        final byte[] message = newUser.getPublicKey().getEncoded();
+        final byte[] signature = ED25519.generateSignature(message, owner.getPrivateKey());
+        list = API.getBlocks(owner);
+        API.addAcquaintanceToChain(newUser, signature, message);
+        assertNotEquals(API.getBlocks(owner), list);
```

Comment:
I still find the getBlocks not a clear name. For other teams that will use our API. They are interested in the users, right? Can't you rename it to 'getContacts' or something?